### ISSUE TITLE
Releasing version 3.0?

### ISSUE THREAD

**Issue by ShaiNahum** [GitTools/GitLink#161]
So, what are the major tasks before releasing version 3.0?

I want to utilize this awesome tool in my project - and I require features that exist only in the development branch. I rather rely on a stable branch in my code - mainly because it's much easier to convince my teammates to approve it if it doesn't contain the word 'unstable'. I'll be happy to help with any important tasks

**Comment by GeertvanHorrik** [comment]
There is nothing really holding this back. It's probably ready for a stable release, but might break quite some builds if people blindly install the latest version (hence our hesitation to release this as stable).

**Comment by GeertvanHorrik** [comment]
If you can test whether the latest unstable works for you, I am happy to release this later this week as stable.

**Comment by ShaiNahum** [comment]
Thanks for the super fast reply!
 I'll check this 👍

**Comment by ShaiNahum** [comment]
tl;dr - Not working for me, but might not be related to GitLink. If I don't respond within a reasonable time, don't wait for me.
--
The source files do not load for some reason. When I try to look into it, the xxx.srcsrv file looks good - but the pdb file looks weird. The srcsrv section inside it is sometimes duplicated and at other times cut in the middle. I'm not sure if its related to the problem I'm facing. I'll try to recreate the problem later this week, using a simpler project that I can share. However, it could be that the problem is related to the weird build system my team is using and is not related to GitLink at all.

**Comment by ShaiNahum** [comment]
Works! I had a config problem on my side.

The resulting PDB file was perfectly ok. I had some unrelated problem with visual studio / tf.exe.
Let me know when you release the version as stable :)

**Comment by GeertvanHorrik** [comment]
Nice, thanks for the feedback. We'll release a new unstable now since a new PR has just been accepted. Please make sure to test it and let us know whether that one works.

**Comment by ShaiNahum** [comment]
Sure! Will test it first thing next week. I want to make sure that the 'UnescapeDataString' doesn't act funny on our remote build system.

**Comment by ShaiNahum** [comment]
It works well :)

**Comment by GeertvanHorrik** [comment]
Today is the day. Closing this ticket now.
